pandas/core/internals/blocks.py

```diff
@@ -1260,12 +1260,12 @@ def diff(self, n, axis=1):
         new_values = algos.diff(self.values, n, axis=axis)
         return [self.make_block(values=new_values)]
 
-    def shift(self, periods, axis=0):
```

Comment:
add to ExtensionBlock.shift, and any other subclasses. It should be passed through to the underlying `self.values.shift`

Reply:
added